crates/hyperswitch_connectors/src/utils.rs

```diff
@@ -1638,6 +1638,10 @@ pub trait ForeignTryFrom<F>: Sized {
     fn foreign_try_from(from: F) -> Result<Self, Self::Error>;
 }
 
+pub trait ForeignFrom<F> {
+    fn foreign_from(from: F) -> Self;
+}
+
```

Comment:
Remove this trait and instead of using `foreign_from` create a function instead in `crates/router/src/connector/forte/transformers.rs`